testnet/stacks-node/src/tests/neon_integrations.rs

```diff
@@ -1915,6 +1915,191 @@ fn microblock_integration_test() {
     channel.stop_chains_coordinator();
 }
 
+#[test]
+#[ignore]
+fn filter_low_fee_tx_integration_test() {
```

Comment:
Should these new tests be added to the integrations dockerfile?

Reply:
Done